static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -151,6 +152,14 @@ const updateHistory = (): void => {
     }
 };
 
+const updateDailyArticleCount = (): void => {
+    const page = config.get('page');
+
+    if (page) {
```

Comment:
Double checking that this function should be called for every `page`. I'm pretty sure this is distinct from a front, but is it definitely an article?

Reply:
this `page` could be a front, but we check that it's an article inside `incrementDailyArticleCount`

Reply:
👍